src/components/AccountCard.vue

```diff
@@ -76,9 +76,8 @@ export default defineComponent({
         const availableTokens = ref<Token[]>([]);
 
         const stakedRefund = computed((): number =>
-            (accountData.value?.refund_request?.cpu_amount.value +
-             accountData.value?.refund_request?.net_amount.value)
-             ?? 0,
+            (accountData.value?.refund_request?.cpu_amount.value ?? 0) +
+            (accountData.value?.refund_request?.net_amount.value ?? 0),
```

Comment:
this was causing a NaN issue on the account page because `undefined` was being used for arithmetic when balance is 0

Reply:
Yeah, I saw that too and fixed it in my current PR. So the one who merges second will have to solve conflicts XD